src/_shared/scss/_popup.scss

```diff
@@ -0,0 +1,297 @@
+/* ==========================================================================
```

Comment:
Is this file needed? See Regis' comment: https://github.com/guardian/commercial-templates/pull/43